src/CalcViewModel/Common/KeyboardShortcutManager.cpp

```diff
@@ -72,24 +72,30 @@ namespace CalculatorApp
 
         void LightUpButton(ButtonBase^ button)
         {
-            // If the button is a toggle button then we don't need 
+            // If the button is a toggle button then we don't need
             // to change the UI of the button
             if (dynamic_cast<ToggleButton^>(button))
             {
                 return;
             }
 
-            // The button will go into the visual Pressed state with this call 
+            // The button will go into the visual Pressed state with this call
             VisualStateManager::GoToState(button, "Pressed", true);
 
+            // This timer will fire after c_lightUpTime and make the button
```

Comment:
It's confusing to see this being added here.  Is this just a result of an out-of-date branch?

Reply:
I believe, so. I fixed it.